doc/string/codepoints.rdoc

```diff
@@ -0,0 +1,6 @@
+Returns an array of the codepoints (integer ordinals) in +self+;
+see {}[]:
```

Comment:
Is this a typo?

Reply:
No, it's a thinko.  I thought about linking out, but did not find what I wanted.